src/actions/markets.js

```diff
@@ -2,12 +2,11 @@ import {
 	FETCH_SNX_CURRENCY,
 	FETCH_SNX_CURRENCY_PRICE,
 	FETCH_SETH_CURRENCY_PRICE,
+	FETCH_SUSD_CURRENCY_PRICE,
 	FETCH_SNX_CURRENCY_SUCCESS,
-	FETCH_NUSD_CURRENCY,
-	FETCH_NUSD_CURRENCY_SUCCESS,
 } from './actionTypes';
 
-export const fetchSNX = () => ({
+export const fetchSnx = () => ({
```

Comment:
shouldn't we keep it as `fetchSNX`?

Reply:
No, I like this better. The whole codebase has shitty naming conventions I will change over time. it is just a bit of a mess with the naming conventions

Reply:
haha ok.. well you know this repo better.